src/assets/styles/_mixin.scss

```
@@ -33,6 +33,8 @@
     'light-color': $light-background,
     'primary-color': $primary,
     'l-blue': $light-blue,
+    'l-grey': $light-grey,
+    'smoke-bg': $white-smoke,
```

need to refactor these color names. 
```suggestion
  'black-6': $black-color-6,
    'black-38': $black-color-38,
    'white': $white-color,
    'black-87': $black-color-87,
    'yellow': $yellow,
    'faint-blue': $faint-blue,
    'primary': $primary,
    'light-blue': $light-blue,
    'light-grey': $light-grey,
    'white-smoke': $white-smoke,
```

---

ana, 
when i keep color name just as 'yellow', it isnt showing any change. It works only when i keep the color name as 'yellow-color'. I did not understand why.